### PR TITLE
docs-content: Docs related to rolling update strategies

### PR DESCRIPTION
**Pull request by hectorj2f**
Please, take a look.. Do I miss any other file to update ?

@marians @puja108

### CODE REVIEW

File: content/reference/api/index.md [hunk 1]
```diff
@@ -1004,16 +1004,26 @@ curl -sS \
...
 ### Update an Existing Component {#UpdateComponent}
 
-To update a service component's image for an application, call the `POST` method on the `/v1/org/{org}/env/{env}/app/{app}/service/{service}/version/{version}/update` endpoint:
+To update a service component's image for an application, call the `POST` method on the `/v1/org/{org}/env/{env}/app/{app}/service/{service}/version/{version}/update` endpoint.
+You can optionally choose among three different rolling update strategies that will
+change the way this update operation is triggered over your component.
+
+These strategies are:
+
+* `one-for-one(default)` updates one after the other all the instances of your component.
```
Comment (ZeissS) [inline content/reference/api/index.md]: IMHO the default part should not be in the label. Users could think this is part of the indended name.

**Comment by ZeissS** [comment]
We should add a note to the changelog that the component-update route now requires a body with the strategy.

File: content/reference/api/index.md (hunk 1 above)
Comment (puja108) [inline content/reference/api/index.md]: +1. I'll change that

**Comment by puja108** [comment]
@ZeissS is the body required? doesn't it default to one-by-one?

**Comment by puja108** [comment]
updated the wording in some areas, LGTM

**Comment by puja108** [comment]
@hectorj2f just realized the nameing: could we rename to one-by-one instead of one-for-one?

**Comment by hectorj2f** [comment]
@ZeissS @puja108 Added change_log

**Comment by puja108** [comment]
LGTM

**Comment by marians** [comment]
Coming in late, I recognized that the API changelog now has the newest entry at the bottom. I'll turn that around in a new change.

**Pull request merged**